### PR TITLE
Added close bill feature

### PR DESCRIPTION
**Pull request by isabella-rocha**

### CODE REVIEW

File: spec/controllers/bills_controller_spec.rb [hunk 1]
```diff
@@ -216,4 +215,50 @@
...
+        end
+        it 'shoud not close the bill' do
+          get "/bill/close/#{bill.id}"
+          
+          bill.status.should == :waiting_confirmation
+        end
+      end
+    end
+  end
+end
```
Comment (brianstorti) [inline spec/controllers/bills_controller_spec.rb:270]: Please, add a blank line between each code block.

File: controllers/bills_controller.rb [hunk 2]
```diff
@@ -91,3 +91,14 @@
...
+
+get '/bill/close/:bill_id' do
+  @bill = Bill.find(params[:bill_id])
+  if logged_as_admin? && @bill.status == :waiting_confirmation
+    @bill.status = :closed
+    @bill.save!
+    redirect '/', :success => I18n.t("bill_closed_ok")
+  else
+    redirect '/', :error => I18n.t("not_an_admin_account")
+  end
```
Comment (brianstorti) [inline controllers/bills_controller.rb:105]: So if I try to "close" a bill that is not waiting for confirmation I get the "not and admin" error, even if I'm an admin, right?
It'd be nice to have more specific error messages for each case.

File: views/bills/index.erb [hunk 3]
```diff
@@ -63,9 +63,12 @@
                     <button type="submit" class="tiny button" onclick="return confirm('<%= I18n.t("confirm_remove")%>')">
                       <%= I18n.t("remove_bill")%>
                     </button>
+                    <% if bill.status == :waiting_confirmation %>
+                      <a href="/bill/close/<%= bill.id %>" class="tiny button"><%= I18n.t("close_bill") %></a>
+                    <% end %>
                   </form>
                 </div>
-              <% end %>
+              <% end %>
```
Comment (brianstorti) [inline views/bills/index.erb]: Please, remove the whitespaces in the end of this line.

Comment (isabella-rocha) [inline views/bills/index.erb]: :ok:

File: controllers/bills_controller.rb (hunk 2 above)
Comment (isabella-rocha) [inline controllers/bills_controller.rb:105]: Done :-]

**Pull request merged**